plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2497,16 +2385,14 @@ void producer_plugin_impl::process_scheduled_and_incoming_trxs(const fc::time_po
       }
       LOG_AND_DROP();
 
-      incoming_trx_weight += _incoming_defer_ratio;
-
-      if (sch_itr_next == sch_idx.end())
+      if (expired_itr_next == expired_idx.end())
          break;
-      sch_itr = sch_idx.lower_bound(boost::make_tuple(next_delay_until, next_id));
+      expired_itr = expired_idx.lower_bound(boost::make_tuple(next_expired, next_id));
```

Comment:
Don't actually need this `lower_bound` now as the `expired_itr_next` can just be used for `expired_itr`. I'm ok with not updating this and leaving as is if you prefer.